numpy/core/src/multiarray/abstractdtypes.c

```diff
@@ -108,18 +108,21 @@ NPY_NO_EXPORT HPyType_Spec HPyArray_PyIntAbstractDType_spec = {
     .name = "numpy._IntegerAbstractDType",
     .basicsize = sizeof(PyArray_Descr),
     .flags = Py_TPFLAGS_DEFAULT,
+    .builtin_shape = HPyType_BuiltinShape_Legacy
```

Comment:
If there is a `HPyType(_LEGACY)_HELPERS(PyArray_Descr)`, we should use `SHAPE(PyArray_Descr)` here.

Reply:
right, but as far as I can tell there is no such use of the HEPLERS.